lib/model/contact.dart

```diff
@@ -0,0 +1,54 @@
+import 'id.dart';
+import 'location.dart';
+import 'name.dart';
+import 'picture.dart';
+import 'registered.dart';
+import 'dob.dart';
+import 'login.dart';
+
+class Contact {
+  String gender;
+  Name name;
+  Location location;
+  String email;
+  Login login;
+  Dob dob;
+  Registered registered;
+  String phone;
+  String cell;
+  Id id;
+  Picture picture;
+  String nat;
```

Comment:
you could change this for a more descriptive name

Reply:
I didn't know what it was either, I think the nationality, I changed it there